### DISCUSSION TITLE
why github dont take my student Id for github student pack i want to register for octernship

### DISCUSSION THREAD

**Discussion by avdhendra** [community/community#50078] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have tried all the thing like student id fee reciept college lms portal image none of them work what should i do in this case i also tweet about it

**Comment by neutrinomuon** [comment]
Hello,

So, just to make sure. Have you followed all guidelines from https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student ?

#### Requirements

To be eligible for GitHub Global Campus, including GitHub Student Developer Pack and other benefits, you must:

- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, -homeschool, or similar educational institution
- Have a verifiable school-issued email address or upload documents that prove your current student status
- Have a [GitHub personal account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
- Be at least 13 years old

Also, Documents that prove your current student status must include **a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter**. 

During your tenure as a student, you may be prompted to periodically re-verify your current academic status.

I hope this helps,

**Reply by avdhendra** [reply]
Yeah i have the Id of college and the fee reciept where the data of my college ending is there but still it wont let me gave the github student pack

**Comment by neutrinomuon** [comment]
I believe this might not be sufficient. Does your ID of college have a current enrollment date, validity and picture? A free receipt does not seem to be valid as a document. Probably, It is better (more credible) to have an **enrollment verification letter** with a verified signature from your college. Did you also use a verifiable school-issued email address? However, if all these steps were done, maybe you need to contact support. Please, let me know.

**Reply by neutrinomuon** [reply]
Thank you for submitting the documentation. However, I have some feedback regarding the images you provided:

- The first image does not provide sufficient evidence of your enrollment. It only shows the university's webpage, which cannot be considered a valid document.
- The second image is just a payment receipt and cannot be accepted as proof of enrollment. It only shows that you have paid but not that you are currently enrolled as a student. This could potentially create uncertainty regarding your current student status, especially if you have transferred from another school.
- The third image only shows the classes provided by the university and does not include any personal information or evidence of your enrollment status. Even if your name were included, it would not be considered a valid document.
- The fourth image. There seems to be a problem with the student card as full proof. It only shows 2020-23. This could potentially create uncertainty regarding your current student status, particularly if you have already completed your studies or transferred to a different school. 
- The fact that your college did not provide you with a college email ID could pose a significant issue. See one possible solution below. 

Therefore, I strongly recommend that you obtain a signed letter of enrollment from your school coordinator that clearly states your enrollment status and expected graduation dates. This will provide the necessary evidence to verify your student status and ensure that you meet the requirements for the GitHub Student Developer Pack.

**Reply by avdhendra** [reply]
Ok
Thanks for giving me advice

**Reply by neutrinomuon** [reply]
Sure, if my response was helpful to you, I would be happy to have it marked as a valid answer for others in the community who may face a similar issue. Thank you for considering it!

**Reply by avdhendra** [reply]
how to validate you answer i dont know can you tell me

**Reply by neutrinomuon** [reply]
Sure. To mark a response as a valid answer on GitHub, follow these steps:

- Go to the GitHub page where the question was asked and find the response that you want to mark as a valid answer.
- Click on the three dots (ellipsis) in the top-right corner of the response.
- In the dropdown menu that appears, select "Mark as resolved".
- You will be prompted to confirm that you want to mark the response as a valid answer. Click "Mark as resolved" to confirm.
- The response will now be marked as a valid answer, and other users will be able to see that it has been accepted as a solution.

**Reply by Jinsssss333** [reply]
hi..i am facing the same issue too
i merged my physical id cards image into one pic which contains all the given details as a student id..they  didnt accept it twice
